### DISCUSSION TITLE
macOS 32bit bridge enable

### DISCUSSION THREAD

**Discussion by Deftaudio** [falkTX/Carla#1737] [General]
Please, advise how to enable 32bit bridge in macOS? 
I'm using the latest 2.5.3 release. After enabling experimental features it now appears in the plugin scan window but this option is grayed out. 
I also installed it with brew and the problem is the same. It seems carla-discovery-posix32 module is missing. Would you please suggest steps to install it?

Also I tried to compile it from scratch, but I can't compile make posix32 no matter of SDK library used, each has various juce errors.

**Comment by falkTX** [comment]
This is not possible, Apple removed the hability to run 32bit applications from the OS itself.

**Reply by Deftaudio** [reply]
> This is not possible, Apple removed the hability to run 32bit applications from the OS itself.

Do you have the solution for this dated environment? :)

**Comment by Deftaudio** [comment]
I should mention that, I'm on High Sierra and Catalina.
